websites/docs-smoke-test/src/content/release-notes.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Docs Kit Release Notes Test
+# TODO frontmatter to instruct that this is a release note layout? or MDX imports?
```

Comment:
Are release notes supposed to be "integrated" within an existing docs website, like MC?
Or should they "live" in their own website / gatsby project?

In the first case, it would probably make more sense to have a dedicated **folder**, so that the appropriate layout is automatically selected.

In both cases, I'm not sure I would make it configurable via frontmatter (very much error prone).

Reply:
**Separate website or not?**
I checked with stakeholders like POs (and I think even you) and the conclusion was to stick with separate release notes lists per part of our product (i.e. currently MC, API, App Kit, SDKs, ...) instead of a central aggregated one.  We can nevertheless choose to have a special website that contains nothing but certain release notes if we want. 

**Why the content folder?**
We chose to make the URL namespace inside a website 1:1 the folder structure in `src/content` so there is no conflict free namespace or "folder" outside that physical folder.   That's why sticking to that and having at least a placeholder file for the release notes URL felt better to me. 

The text, heading etc. above the release notes list is custom per e.g. MC vs. API vs. others and I like the Idea that it can be easily modified and authored by docs and PO staff like any other page.  That's how bigger component based CMS systems work, too.  Everything is a page, but some consist of just one big component.  

**Frontmatter // how to control the page layout? *** 
I don't really like it, too - just mentioned it because the that's how Jekyll is doing it right now - you put a "layout" frontmatter to choose the page template.  

How do you imagine to author home pages like https://docs.commercetools.com/custom-applications/ in the future?  It's a similar case of a certain page URL having a differing layout.  Apropos, It would be great if that can be MDX, too :-)